### DISCUSSION TITLE
Force password change

### DISCUSSION THREAD

**Discussion by s2b-git** [YetiForceCompany/YetiForceCRM#15821] [General] [answered]
If I change "Force password change" value directly in database it doesn't bring effect, it has to be done from user panel. How to change it for many users (more than 300) en mass?

**Comment by mariuszkrzaczkowski** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/10198654/160337557-6d320625-cb55-41ae-9961-b85b20fe6482.png)
or
![image](https://user-images.githubusercontent.com/10198654/160337681-60897cc2-e9e0-45bf-9688-ab3b86a1b0ae.png)
https://yetiforce.com/pl/component/zoo/item/how-to-reset-a-user-s-password-without-access-to-the-crm-system
